docs/docsite/rst/dev_guide/developing_modules_documenting.rst

```diff
@@ -10,6 +10,88 @@ which conforms to the schema defined below. You may find it easier to
 start writing your ``DOCUMENTATION`` string in an editor with YAML
 syntax highlighting before you include it in your Python file.
 
+All modules must have the following sections defined in this order:
+
+1. ANSIBLE_METADATA
+2. DOCUMENTATION
+3. EXAMPLES
+4. RETURNS
+5. Python imports
+
+.. note:: Why don't the imports go first?
+
+  Keen Python programmers may notice that contary to PEP8's advice we don't put ``imports`` at the top of the file. This is due to how modules used to be combined with libraries. For more information see :doc:`developing_program_flow_modules`.
+
+
+
+ANSIBLE_METADATA Block
+''''''''''''''''''''''
+
+Some into line needed
+
+For new modules the following block can be simply added into your module::
+
+   ANSIBLE_METADATA = {'status': ['preview'],
+                       'supported_by': 'community',
+                       'version': '1.0'}
+
+.. note:: version field
+
+   This is the version of the ``ANSIBLE_METADATA`` schema, *not* the version of the module.
+
+.. note::
+
+   Promoting a modules ``status`` or ``supported_by`` status should only be done by members of the Ansible Core Team.
+
+Version 1.0 of the metadata
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**Structure**::
```

Comment:
Note, formatting of this section currently seems to be a quick port from the markdown (since it was in a github issue).  Probably should use rst section headers and field lists isntead.